slices/dotnet-templates-8.0.yaml

```
@@ -0,0 +1,13 @@
+package: dotnet-templates-8.0
+
+essential:
+  - dotnet-templates-8.0_copyright
+
+slices:
+  libs:
+    contents:
```

```suggestion
  libs:
    essential:
      - dotnet-host-8.0_bins
    contents:
```

From the archive metadata, this package depends on `dotnet-host-8.0`. However, the contents in this package are only of type `Microsoft OOXML`. Feel free to reject this suggestion if this dependency is not necessary.